x/evm/abci.go

```diff
@@ -615,26 +660,21 @@ func handleMessages(ctx sdk.Context, bk types.BaseKeeper, n types.Nexus, m types
 		for _, msg := range msgs {
 			success := false
 			_ = utils.RunCached(ctx, bk, func(ctx sdk.Context) (bool, error) {
-				cmdID, err := handleMessage(ctx, ck, n, m, chain, msg)
-				if err != nil {
+				if err := validateMessage(ctx, ck, n, m, chain, msg); err != nil {
 					return false, err
```

Comment:
maybe add an info log here for the error and add the chain and message id as facets?